plutus-core/cost-model/budgeting-bench/Benchmarks/Nops.hs

```diff
@@ -132,12 +133,12 @@ nopCostParameters =
 infixr >:
 (>:) :: uni ~ DefaultUni
      => SomeConstant uni Integer
-     -> EvaluationResult Integer
-     -> EvaluationResult Integer
+     -> BuiltinResult Integer
+     -> BuiltinResult Integer
```

Comment:
We now recommend using`BuiltinResult` instead of `EvaluationResult` for all builtins. The latter doesn't allow for attaching any error messages to errors, so it simply shouldn't be used (it's now a type error to use `EvaluationResult` in a builtin).